htdocs/contact/class/contact.class.php

```diff
@@ -750,7 +750,8 @@ public function fetch($id, $user = null, $ref_ext = '', $email = '')
 				$this->entity			= $obj->entity;
 				$this->ref				= $obj->rowid;
 				$this->ref_ext			= $obj->ref_ext;
-
+                
+                $this->civility_id      = $obj->civility_code;
```

Comment:
If we do not include this line, does this create a bug somewhere ?

Reply:
@eldy for the line 754 no but i need the other in API. i just paste the same as member or user class for more standardization

Reply:
We try to standardize properties with this rule for foreign properties on another dictionary table :
_id is the id
_code is the code
_label or nothing is the label

That's why i prefer having no civility_id set, because we don't know the rowid and it is not the code and we should encourage to use the code.